api/client.yaml

```diff
@@ -1705,6 +2017,12 @@ components:
       enum:
         - individual
         - business
+    OwnerType:
+      type: string
+      description: The type of individual owner of a resource, such as SSN, phone, or address
```

Comment:
How can an SSN, phone, or address own an address? 

What's this ownerTyped used by? Why is it needed?

Reply:
An OwnerType would be either a customer or a representative.  I'll fix the description

Reply:
I don't understand why those models need `OwnerType`? A Customer or CustomerRepresentative would have an SSN, phone, etc. Why the back reference?

Reply:
It's needed because we now have two different types that can have a SSN, phone, or address - `1) a customer` or `2) a customer representative` for business customers

Reply:
I don't understand why the API needs to know or care about that though? Can't we join on `ssn`, `phones`, etc having a customer_id or representative_id if we're going to merge the `customer_ssn` table.

Typically in the past I would have made a `representative_ssn` table that's separate so we didn't have to make the clients aware of implementation details.

Reply:
I see what you're saying.  The reason I made the decision to add an owner type field, is that I was worried about the possibility of representative ID and customer ID collisions

Reply:
To your other point, I considered just adding new tables for `representative_address`, `representative_phone`, and `representative_ssn`, but I felt like those should be shared resources since they're exactly the same thing, and I wanted to avoid a lot of duplicated logic everywhere.  Anytime we make a change to address or ssn or phone, we would have had to make that change in two different places, which didn't seem ideal in my mind

Reply:
I've duplicated these types of tables before and it's been okay. Typically after the initial implementation there wasn't much to modify in both places. I understand not wanting to duplicate all that repository code though, but copy/paste seems a lot easier now. 

I didn't realize you had implemented all of this already. I wanted to get the API agreed on before we started down the coding route, so now I feel bad if you backtrack on a bunch of this work.

Reply:
I figured it could require changes.  And Dylan is looking to have something to test on his end, so I wanted to get most of it completed.  Hopefully, it won't require changing the world

Reply:
There are other customers besides our frontend team though. It's costly to break API endpoints like this for folks outside of Moov. They're greeted with unexpected development time from upgrading.

Reply:
I removed the `OwnerType` property from all of the address requests in my latest commit @adamdecaf